src/components/Home/components/BoardsHome/BoardsHome.js

```diff
@@ -33,12 +34,13 @@ const BoardsQuery = gql`
 `
 
 const BoardsHome = props => {
+  const CreateBoardPopOver = PopOver(CreateBoard, CreateBoardButton)
   return (
     <div>
       <Query query={BoardsQuery} variables={{ id: props.userId }}>
         {({ loading, data }) => {
           return (
-            !loading && data && (
+            !loading && data && data.User && data.User.boards && (
```

Comment:
I think that check `data.User.boards` is not necessary.
- Have you try only with `data.User`?

`!loading && data.User && (...)`

Reply:
I was a little paranoid with this. I will go ahead and refactor these.

Reply:
@nickeyvee That's ok. Try it.